### ISSUE TITLE
Accessibility: Screen reader is not announcing change in expanded/collapsed state of PropertyGrid Category item

### ISSUE THREAD

**Issue by M-Lipin** [dotnet/winforms#1746]
*Repro steps:*
1. Create and launch .NET Core WinForms application project with PropertyGrid control with properties bound to some object/control.
2. Start Screenreader.
3. Navigate till any Category and hit enter to change its expanded/collapsed state.
 
*Actual:*
When navigated to any Category inside table for example, Accessibility & it is in expanded state then Narrator will announce it as expanded, but if we hit enter to collapse it, narrator is not announcing its new state which is collapsed.
Note: this issue is common for both Narrator & NVDA.
 
*Expected:*
Screen reader should announce current state of Category accordion whether it is expanded or collapsed (changing state should trigger announcement).
 
*User Impact:*
Screen reader users will not get to know that if Category accordion is expanded or collapsed.

DevDiv issue: https://devdiv.visualstudio.com/DevDiv/_workitems/edit/956737

**Comment by Jessie-Zhang01** [comment]
Verified this bug with .Net Core 3.0.100-014265 from Release branch + Winforms designer VSIX(https://dev.azure.com/dnceng/internal/_build/results?buildId=356547), this bug is fixed. 

- In Narrator, because of Inspect accessibility property change, items that support collapse/expand behavior can be announced correct "collapse/expand" state in Narrator when collapse or expand Property entries/ items. And can be announced correct position info: xxx of xxx.

- In Inspect, when categorized to sort PropertyGrid item, the ControlType of top-level items and all sub-items are changed to TreeItem
Please see the below screenshots:
![image](https://user-images.githubusercontent.com/45864649/65032126-2237e400-d975-11e9-8a2b-9916556bfbd1.png)
![image](https://user-images.githubusercontent.com/45864649/65032138-29f78880-d975-11e9-8172-cdb89658ee76.png)
![image](https://user-images.githubusercontent.com/45864649/65032155-3380f080-d975-11e9-88e7-c4ab6c82ddea.png)
